bindings/python/fdb/impl.py

```diff
@@ -1341,7 +1341,7 @@ def create_transaction(self):
         return Transaction(pointer.value, self)
 
     def get_id(self):
-        if self.idFuture == None:
+        if self.idFuture == None and self.tpointer != None:
```

Comment:
I don't think we typically check whether the pointer is nulled out in these objects. In theory that shouldn't be possible unless the user is doing something weird.

Reply:
I guess this is also potentially a thread safety concern, so we could try to introduce a lock. Maybe easier and better for now is to just go back to what you had and then we can add support for this later if needed.